### ISSUE TITLE
Doc update needed for `cool_off` template context variable

### ISSUE THREAD

**Issue by nealtodd** [jazzband/django-axes#428]
After v4.5.4 the template context variable name for the cool off time changed from `cooloff_time` to `cool_off`:

https://github.com/jazzband/django-axes/commit/a4c4ba6fb73d3dc5a07fed499bc051172f21a3c4#diff-44933db6f07215a8e2115342b928a9c5R300

The configuration documentation in the run up to v5.0 for `AXES_LOCKOUT_TEMPLATE` still refers to `cooloff_time`:

https://github.com/jazzband/django-axes/blob/master/docs/4_configuration.rst#configuring-project-settings

**Comment by aleksihakli** [comment]
Thanks for the report! Seems like the name change might be unnecessary and break backwards compatibility for templates. It could be a good idea to revert back to the old name so that peoples old templates and setups work smoothly in 5.0.0

**Comment by nealtodd** [comment]
Yes, it could revert - there didn't look to be any particular reason for it to have changed.

Alternatively, it could be included in a list of tweaks for migrating from 4.x to 5.x because there are a few other backwards incompatible changes to make, e.g.

- `AxesModelBackend` renamed to `AxesBackend` for `AUTHENTICATION_BACKENDS`.
- Addition of `axes.middleware.AxesMiddleware` to `MIDDLEWARE`.
- `axes_reset_user` management command renamed to `axes_reset_username`.

(along with `cooloff_time` to `cool_off` these were the changes I needed to make when bumping from 4.5.4.)

**Comment by aleksihakli** [comment]
I addressed the changes and will open a PR for them.

- I added a deprecation shim for the `AxesModelBackend` that should support the previous version and imports as well.
- Checks were added for the `MIDDLEWARE` being misconfigured or in other words `axes.middleware.AxesMiddleware` missing from configuration.
- `axes_reset_user` has been added as a symlink for backwards compatibility.

**Comment by aleksihakli** [comment]
@nealtodd version 5.0b1 has now been released to PyPI and should address the issues discussed in here. If you have the previous upgrade environment available, would you mind testing if everything except the middleware works as intended?

**Comment by nealtodd** [comment]
Updating to 5.0b1 from the checkout that was using 4.5.4 gave the expected checks errors. After changing the backend name and adding the middleware the checks errors disappeared also as expected. Everything tested fine then on 5.0b1.

The only thing to note is that the PyPI 5.0b1 package won't return any checks errors via `python manage.py check` because efe4e960b landed on master just after it so the `axes.checks` aren't registered (I patched it in on my local install of 5.0b1 to confirm that they are reported with change).

The next 5.0x package from master will of course resolve that.

Thanks again @aleksihakli for the quick updates.

**Comment by aleksihakli** [comment]
@nealtodd I baked the final production release 5.0. The final has the `python manage.py check`s enabled and has packaging and documentation improvements. I'll try and get the documentation updated for 5.0 on RTD soon as well. I don't personally have permissions on the RTD site, but I've contacted @camilonova and @jezdez for the update.

Thanks for your feedback and help with prepping the final release!
